<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBackupUnitDlg</class>
 <widget class="BaseDlg" name="AddBackupUnitDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">AddBackupUnitDlg</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>290</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>290</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>400</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>备选单位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="unit_edit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>40</height>
       </size>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>单位倍率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="factor_edit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>40</height>
       </size>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>AddBackupUnitDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
